tutorials/editor_basics/coordinates_system.md

```diff
@@ -1,20 +1,31 @@
 ---
-title: 2. Editor coordinates system. Positions and offsets
+title: Editor Coordinate Systems - Caret Positions and Offsets
 ---
 
+The previous tutorial [Working with Text](working_with_text.md) discussed extending the [AnAction](upsource:///platform/editor-ui-api/src/com/intellij/openapi/actionSystem/AnAction.java) class, and using an [AnActionEvent](upsource:///platform/editor-ui-api/src/com/intellij/openapi/actionSystem/AnActionEvent.java) object.
+The event object provides access to [Project](upsource:///platform/core-api/src/com/intellij/openapi/project/Project.java), [Document](upsource:///platform/core-api/src/com/intellij/openapi/editor/Document.java), and [Editor](upsource:///platform/editor-ui-api/src/com/intellij/openapi/editor/Editor.java) objects.
```

Comment:
why do we need 2nd paragraph here?

Reply:
Just a review, as on editor_events. I'll shorten it.

Reply:
Shortened, now less specific, more a general overview.